tests/tests_app/core/test_lightning_flow.py

```diff
@@ -3,21 +3,24 @@
 from collections import Counter
```

Comment:
There are many missing tests. For example, we wouldn't want to allow users to register commands of methods defined in the base class, e.g., run, configure_layout, etc. This would lead to extreme misbehavior.